containers/cpp-mariadb/test-project/main.cpp

```diff
@@ -0,0 +1,68 @@
+/*-------------------------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See https://go.microsoft.com/fwlink/?linkid=2090316 for license information.
+ *-------------------------------------------------------------------------------------------------------------*/
+
+#include <iostream>
+#include <mariadb/conncpp.hpp>
+
+using namespace std;
+
+int main()
+{
+    cout << "Hello, Remote World!" << endl;
+
+    try
```

Comment:
is this the right place for the `try`? It seems a little generous since getenv for sure doesn't throw. The line with `sql::mariadb::get_driver_instance` seems like the right place.

Reply:
Great comments @xtophs. I'll take care of `main.cpp`

Reply:
hi @juzuluag, my understanding is xtophs recommended we move the `try` to right before the line `sql::Driver* driver = sql::mariadb::get_driver_instance`, rather than remove it altogether.